ui/react-components/components/Dropdown/AsyncDropdown.test.jsx

```diff
@@ -0,0 +1,214 @@
+import {fireEvent, waitForElement} from "@testing-library/react";
+import React from "react";
+import AsyncDropdown from "./AsyncDropdown.jsx";
+import selectEvent from "react-select-event";
+import '@testing-library/jest-dom/extend-expect';
+import {renderWithReactIntl} from '../../utils/TestUtil';
+
+const loadOptions = (inputValue, callback) => {
+    // eslint-disable-next-line angular/timeout-service
+    setTimeout(() => {
```

Comment:
Have we tried `useTimeout`? https://github.com/streamich/react-use/blob/master/docs/useTimeout.md